js/packages/teams-ai/src/TurnStateProperty.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @module teams-ai
```

Comment:
I'm a little concerned this file is superfluous. TurnState.ts already has `deleteValue`, `hasValue`, `getValue`, and `setValue`. 

Is there a difference in functionality because this is using `any` instead of `unknown`?

Reply:
(Per our 1:1 call) Here could you leave a comment that this is a wrapper specifically for Dialogs, since it needs `StatePropertyAccessor`? TY!

Reply:
done